File: service/jvm-java-quarkus/src/main/java/com/ecobenchmark/controllers/addlist/ListResponse.java
package com.ecobenchmark.controllers.addlist;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.time.Instant;
import java.util.UUID;

public class ListResponse {

    private UUID id;

    private String name;

    @JsonProperty("creation_date")
    Instant creationDate;

    public ListResponse(UUID id, String name, Instant creationDate) {
        this.id = id;
        this.name = name;
        this.creationDate = creationDate;
    }

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Instant getCreationDate() {
        return creationDate;
    }

    public void setCreationDate(Instant creationDate) {
        this.creationDate = creationDate;
    }
}

File: service/jvm-kotlin-spring/src/main/kotlin/com/ecobenchmark/kotlinspringjpa/controllers/addlist/ListResponse.kt
package com.ecobenchmark.kotlinspringjpa.controllers.addlist

import com.fasterxml.jackson.annotation.JsonProperty
import java.time.Instant
import java.util.*

data class ListResponse(
    val id: UUID,
    val name: String,
    @JsonProperty("creation_date")
    val creationDate: Instant
)